.circleci/config.yml

```diff
@@ -0,0 +1,27 @@
+version: 2
+jobs:
+  build:
+    parallelism: 1
+    docker:
+      - image: circleci/elixir:1.10
+        environment:
+          MIX_ENV: test
+          DB_HOST: localhost
+          DB_USER: postgres
+          DB:PASSWORD: postgres
+          DB_NAME: xrt_test
+      - image: circleci/postgres:12-alpine
+        environment:
+          POSTGRES_USER: postgres
+          POSTGRES_DB: xrt_test
+          POSTGRES_PASSWORD: postgres
+    working_directory: ~/app
+    steps:
+      - checkout
+      - run: mix local.hex --force
+      - run: mix local.rebar --force
+      - run: mix deps.get
+      - run: mix compile --warnings-as-errors
+      - run: dockerize -wait tcp://localhost:5432 -timeout 1m
+      - run: mix ecto.migrate
+      - run: mix test
```

Comment:
we can also add `mix dialyzer` and `mix credo` here

Reply:
added

Reply:
dialyzer is not found

Reply:
do you have idea why it is failing ? https://app.circleci.com/pipelines/github/retro-tool/retro-tool-api/10/workflows/6d8eab7e-ddab-40fc-a593-cc6322faa34b/jobs/10

Locally dialyzer works

Reply:
Got it! I'd say we need to add it also for the test environment here: https://github.com/retro-tool/retro-tool-api/blob/master/mix.exs#L72

Reply:
Let's see if that fixes the problem

Reply:
That fixed the problem